crates/marketplace/src/service.rs

```diff
@@ -290,43 +66,64 @@ pub struct ProxyGlobalState<Types: NodeType, H: BuilderHooks<Types>> {
 
     // Maximum time allotted to wait for bundle before returning an error
     api_timeout: Duration,
+
+    maximize_txn_capture_timeout: Duration,
```

Comment:
Nit: I feel like using the verb `maximize` is a bit strange, maybe `maximum` or just `max`.

Reply:
It's not really about the timeout here, it's about the process of "maximizing txn capture". Nonetheless, it doesn't look like a good variable name, I've renamed it and added a doc comment for clarity